chain/client/src/stateless_validation/partial_witness/partial_witness_tracker.rs

```diff
@@ -104,13 +105,7 @@ impl CacheEntry {
 
         // Check if the part is already present.
         if self.parts[part_ord].is_some() {
-            tracing::warn!(
-                target: "client",
-                ?shard_id,
-                ?height_created,
-                ?part_ord,
-                "Received duplicate or redundant partial state witness part."
-            );
+            log_assert_fail!("Received duplicate or redundant partial state witness part. shard_id={shard_id:?}, height_created={height_created:?}, part_ord={part_ord:?}");
```

Comment:
nit: is this line too long for lint? I don't think auto linting happens for macros for some reason.

Reply:
yeah `cargo fmt` is not touching them.